openapi/bundle_test.go

```diff
@@ -172,3 +172,77 @@ func TestBundle_SiblingDirectories_Success(t *testing.T) {
 	// Compare the actual output with expected output
 	assert.Equal(t, string(expectedBytes), string(actualYAML), "Bundled document should match expected output")
 }
+
+func TestBundle_AdditionalOperations_Success(t *testing.T) {
```

Comment:
if possible I would try and add the additionalOperations test in to the original test above and update the expected doc etc, to make sure everything is working together.

Generally also with tests like this its better to be asserting the exact shape of the document like we do in the test above rather than all the assertions you have below to catch where we might be introducing unintended changes etc, but that is just another reason to integrate it into the test above (which will then also mean we have the inline tests also testing it)